Error on converted files that do not have XMP

nunomsfranco opened the pull request:
- Reduced job timeout
- Better error catch and logging
- Added functionality for other filetype conversions
- Fixed conversion of png with transparency

nunomsfranco committed:
Error on converted files that do not have XMP
- Reduced job timeout
- Better error catch and logging

numsu commented on docsbox/docs/tasks.py:
```diff
@@ -41,27 +42,33 @@ def create_tmp_file_and_get_mimetype(original_file, filename, stream=False, dele
     if (meta["mimetype"] == "application/pdf"):
-        os.system("ocrmypdf --tesseract-timeout=0 --optimize 0 --skip-text {0} {1}".format(path, output_path))
+        # faster conversion and file size smaller, if XMP is not found a more forcefull conversion is made so file ends as a PDF/A
+        os.system("ocrmypdf --tesseract-timeout=0 --optimize 0 --redo-ocr --skip-big 30 {0} {1}".format(path, output_path))
```
What's the redo-ocr param for? 30 megapixel limit for images it still quite high, don't you think?

nunomsfranco replied:
yeah it should be --skip-text I was testing with redo to see if it was better but there is no change and it takes a bit more.
At what size should it skip the image? this is mainly for performance and I think only needed in the --force-ocr.

numsu replied:
Ok. If it's only for the ones which have no XMP, we should make it so. Also then I think we could keep the 30 since it's not so common

numsu commented on docsbox/docs/utils.py:
```diff
@@ -131,3 +131,14 @@ def remove_XMPMeta(file):
     xmpfile.close_file()
+
+def has_XMP(file):
+    xmpfile = XMPFiles( file_path=file)
```
Extra space

numsu commented on docsbox/docs/views.py:
```diff
@@ -1,43 +1,50 @@
-    app.errlog.log(error_level, message, extra={ "request": request, "status": str(error_code) })
-    flask_abort(error_code, message=message)
+    app.logger.log(error_level, message, extra={ "request": request, "status": str(status_code) })
+    response = jsonify({'status': status_code, 'message': message })
```
Missing space

numsu commented:
Also check the tests :)

nunomsfranco committed:
Test fix
- Added other filetype conversions
- Fixed conversion of png with transparency
- Other small fixes

nunomsfranco committed:
fix

nunomsfranco committed:
test fix
